index.js

```diff
@@ -124,6 +124,7 @@ function generateIssue (body, url, options) {
   var now = new Date()
 
   var description = [
+    options.mention ? options.mention : '',
```

Comment:
In the long run it might be better to just allow to configure the whole text at all :thinking: